src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -852,7 +857,7 @@ public void calculateMaxCitizens()
     {
         int newMaxCitizens = Configurations.maxCitizens;
 
-        for (Building b : buildings.values())
+        for (AbstractBuilding b : buildings.values())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'b' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)